src/ci/scripts/free-disk-space.sh

```diff
@@ -0,0 +1,218 @@
+#!/bin/bash
+# Free disk space on Linux GitHub action runners
+
+# # ======
+# MACROS
+# ======
+
+# macro to print a line of equals
+# # (silly but works)
+printSeparationLine() {
+    str=${1:=}
+    num=${2:-80}
+    counter=1
+    output=""
+    while [ $counter -le "$num" ]
+    do
+        output="${output}${str}"
+        counter=$((counter+1))
+    done
+    echo "${output}"
+}
+
+# macro to compute available space
+# REF: https://unix.stackexchange.com/a/42049/60849
+# REF: https://stackoverflow.com/a/450821/408734
+getAvailableSpace() { echo $(df -a $1 | awk 'NR > 1 {avail+=$4} END {print avail}'); }
```

Comment:
```suggestion
getAvailableSpace() { echo $(df -h $1 | awk 'NR > 1 {avail+=$4} END {print avail}'); }
```

I found `-h` more humanly readable :)